Core/GDCore/IDE/WholeProjectRefactorer.cpp

```diff
@@ -333,9 +340,25 @@ void WholeProjectRefactorer::ApplyRefactoringForVariablesContainer(
       project, eventsVariableInstructionTypeSwitcher);
 }
 
+// TODO Apply the refactor to external layouts.
+void WholeProjectRefactorer::ApplyRefactoringForObjectVariablesContainer(
+    gd::Project &project, gd::VariablesContainer &objectVariablesContainer,
+    gd::InitialInstancesContainer &initialInstancesContainer,
+    const gd::String &objectName, const gd::VariablesChangeset &changeset,
+    const gd::SerializerElement &originalSerializedVariables) {
```

Comment:
It doesn't apply the refactor on external layout because we don't have the abstraction for it.